### PR TITLE
Add can-ndjson-stream to ecosystem docs

### PR DESCRIPTION
**Pull request by bgando**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,6 +65,7 @@
     "can-map-backup": "3.0.3",
     "can-map-define": "3.0.9",
     "can-namespace": "1.0.0",
+    "can-ndjson-stream": "file:../can-ndjson-stream",
```
Comment (nlundquist) [inline package.json]: should probably change this to a published version number before merging

File: package.json [hunk 2]
```diff
@@ -39,12 +39,13 @@
     "github": "https://github.com/canjs/canjs"
   },
   "dependencies": {
+    "bit-docs-generate-html": "^0.6.0",
```
Comment (nlundquist) [inline package.json]: not sure this is needed since it's part of the bit-docs dependencies

File: package.json [hunk 3]
```diff
@@ -95,7 +97,7 @@
     "can-zone": "0.6.7"
   },
   "devDependencies": {
-    "bit-docs": "^0.0.7",
+    "bit-docs": "0.0.7",
```
Comment (chasenlehara) [inline package.json]: Why was this changed?

**Comment by chasenlehara** [comment]
Closing this in favor of https://github.com/canjs/canjs/pull/3370

**Comment by bgando** [comment]
This is a standalone module that #3370 (can-connect-ndjson) depends on but is not the same
